frontend/pages/hosts/details/cards/MunkiIssues/MunkiIssuesTableConfig.tsx

```diff
@@ -0,0 +1,153 @@
+  };
+}
+
+interface IStringCellProps extends ICellProps {
+  cell: {
+    value: string;
+  };
+}
+
+interface IDataColumn {
+  title: string;
+  Header: ((props: IHeaderProps) => JSX.Element) | string;
+  accessor: string;
+  Cell: (props: IStringCellProps) => JSX.Element;
+  disableHidden?: boolean;
+  disableSortBy?: boolean;
+  disableGlobalFilter?: boolean;
+  sortType?: string;
+  // Filter can be used by react-table to render a filter input inside the column header
+  Filter?: () => null | JSX.Element;
+  filter?: string; // one of the enumerated `filterTypes` for react-table
+  // (see https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js)
+  // or one of the custom `filterTypes` defined for the `useTable` instance (see `DataTable`)
+}
+
+interface IMunkiIssueTableData extends IMunkiIssue {
+  time: string;
+}
+
+export const generateMunkiIssuesTableData = (
+  munkiIssues: IMunkiIssue[] | undefined
+): IMunkiIssueTableData[] => {
+  if (!munkiIssues) {
+    return [];
+  }
+  return munkiIssues.map((i) => {
+    return {
+      ...i,
+      time: i.created_at,
+    };
+  });
+};
+
+// NOTE: cellProps come from react-table
+// more info here https://react-table.tanstack.com/docs/api/useTable#cell-properties
+export const generateMunkiIssuesTableHeaders = (): IDataColumn[] => {
+  const tableHeaders: IDataColumn[] = [
+    {
+      title: "Issue",
+      Header: (headerProps: IHeaderProps): JSX.Element => {
+        const titleWithToolTip = (
+          <TooltipWrapper
+            tipContent={`
+            Issues reported the last time Munki ran on each host.
+          `}
+          >
+            Issue
+          </TooltipWrapper>
+        );
+        return (
+          <HeaderCell
+            value={titleWithToolTip}
+            isSortedDesc={headerProps.column.isSortedDesc}
+          />
+        );
+      },
+      disableSortBy: false,
+      accessor: "name",
+      Cell: (cellProps: IStringCellProps) => (
+        <TruncatedTextCell value={cellProps.cell.value} />
+      ),
+      sortType: "caseInsensitive",
+    },
+    {
+      title: "Type",
+      Header: "Type",
+      disableSortBy: true,
+      accessor: "type",
+      Cell: (cellProps: IStringCellProps) => (
+        <TextCell
+          value={
+            cellProps.cell.value.charAt(0).toUpperCase() +
+            cellProps.cell.value.slice(1)
+          }
+        />
+      ),
+    },
+    {
+      title: "Time",
+      Header: (headerProps: IHeaderProps): JSX.Element => {
+        const titleWithToolTip = (
+          <TooltipWrapper
+            tipContent={`
+            The first time Munki reported this issue.
+          `}
+          >
+            Time
+          </TooltipWrapper>
+        );
+        return (
+          <HeaderCell
+            value={titleWithToolTip}
+            isSortedDesc={headerProps.column.isSortedDesc}
+          />
+        );
+      },
+      disableSortBy: false,
+      accessor: "time",
```

Comment:
I think you can just use `created_at` as the accessor here which should eliminate the need for `generateMunkiIssuesTableData`.

Reply:
How embarassing, good catch!